doc/rm/c_cpp_coding_style.md

```diff
@@ -145,6 +145,27 @@ int create_rainbow(int pots_of_gold, int unicorns);
 
 C++ code should target C++14.
 
+### Aggregate Initializers
+
+***C++20-style designated initializers are permitted in C++ code, when used with POD types.***
+
+While we target C++14, both GCC and Clang allow C++20 designated initializers in C++14-mode, as an extension:
+```
+struct Foo { int a, b; };
+
+Foo f = { .a = 1, .b = 42, };
+```
+
+This feature is fairly mature in both compilers, though it varies from the C11 variant in two ways important ways:
+  - It can only be used with structs and unions, not arrays.
+  - Members must be initialized in declaration order.
+
+Because it is espcially useful with types declared for C, we allow designatued initializers whenever the type is a plain-old-data type:
```

Comment:
`espcially` -> `especially`, `type:` -> `type, and:`

Reply:
Done.